package.json

```diff
@@ -150,13 +152,12 @@
     "lint": "npm-run-all --parallel lint:*",
     "lint:css": "stylelint \"**/*.(js|css)\"",
     "lint:css:fix": "npm run lint:css -- --fix",
-    "lint:js": "eslint .",
+    "lint:js": "eslint --ext .js,.md .",
     "lint:js:fix": "npm run lint:js -- --fix",
+    "lint:js:report": "eslint --output-file build/lint-js-report.json --format json .",
     "lint:php": "vendor/bin/phpcs",
     "lint:php:fix": "vendor/bin/phpcbf",
-    "lint:md": "npm-run-all --parallel lint:md:*",
-    "lint:md:js": "eslint **/*.md",
-    "lint:md:docs": "markdownlint **/*.md",
+    "lint:md": "markdownlint .",
     "storybook": "start-storybook --quiet",
     "test": "npm-run-all --parallel test:*",
     "test:js": "jest --config=tests/js/jest.config.js",
```

Comment:
Not that important but could we by perhaps add a new line for running tests with `-- --coverage && open build/logs/lcov-report/index.html`? E.g. `test:js:coverage` or sth similar.

Reply:
Done!